### PR TITLE
Remove all snowflake references in python 2

### PR DESCRIPTION
**Pull request by hithwen**
Integrations core PR https://github.com/DataDog/integrations-core/pull/10105 removes python 2 support for snowflake. This PR removes all omnibus references to it on python 2

### CODE REVIEW

**Review by FlorianVeaux — APPROVED** [review]

**Review by KSerrania — DISMISSED** [review]
LGTM if builds pass

**Comment by FlorianVeaux** [comment]
I think you'll need to remove: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/data/agent_requirements.in#L89

**Comment by hithwen** [comment]
I guess builds cannot pass until the integrations PR is merged

**Review by KSerrania — APPROVED** [review]

**Pull request merged**
